packages/nimble-components/src/wafer-map/index.ts

```diff
@@ -0,0 +1,64 @@
+import { attr } from '@microsoft/fast-element';
+import { DesignSystem, FoundationElement } from '@microsoft/fast-foundation';
+import { template } from '../theme-provider/template';
+import { styles } from './styles';
+import {
+    WaferMapColorBy,
+    WaferMapColorsScale,
+    WaferMapDataType,
+    WaferMapDie,
+    WaferMapOrientation,
+    WaferMapQuadrant
+} from './types';
+
+declare global {
+    interface HTMLElementTagNameMap {
+        'nimble-wafer-map': WaferMap;
+    }
+}
+
+/**
+ * A nimble-styled WaferMap
+ */
+export class WaferMap extends FoundationElement {
+    @attr({
+        attribute: 'wafer-quadrant'
+    })
+    public waferQuadrant!: WaferMapQuadrant;
+
+    @attr({
+        attribute: 'wafer-orientation'
+    })
+    public waferOrientation!: WaferMapOrientation;
+
+    @attr({
+        attribute: 'max-characters'
+    })
+    public maxCharacters!: number;
+
+    @attr({
+        attribute: 'wafer-datatype'
```

Comment:
The other parts of this API consider "data" and "type" to be separate words so lets keep the same convention for the attribute name too.
```suggestion
        attribute: 'wafer-data-type'
```

Reply:
Updated